frontend/test/lit/test_quantum_control.py

```diff
@@ -51,13 +52,23 @@ class CustomQJITDevice(QJITDevice):
         observables = observables_copy
 
         # pylint: disable=too-many-arguments
-        def __init__(self, shots=None, wires=None, backend=None):
-            super().__init__(config, wires=wires, shots=shots, backend=backend)
-
-        def apply(self, operations, **kwargs):  # pylint: disable=missing-function-docstring
+        def __init__(
+            self, shots=None, wires=None, backend_name=None, backend_lib=None, backend_kwargs=None
+        ):
+            self.backend_name = backend_name if backend_name else "default"
+            self.backend_lib = backend_lib if backend_lib else "default"
+            self.backend_kwargs = backend_kwargs if backend_kwargs else ""
+            super().__init__(wires=wires, shots=shots)
+
+        def apply(self, operations, **kwargs):
             pass
 
-    return CustomQJITDevice(wires=num_wires)
+        @staticmethod
+        def get_c_interface():
+            """Location to shared object with C/C++ implementation"""
+            return get_lib_path("runtime", "RUNTIME_LIB_DIR") + "/libdummy_device.so"
```

Comment:
```suggestion
            return "DummyDevice", get_lib_path("runtime", "RUNTIME_LIB_DIR") + "/libdummy_device.so"
```

I think this is not being tested. The `get_c_interface` method changed APIs a while back and instead of returning just the path, it should return the name and the path.

Reply:
Done